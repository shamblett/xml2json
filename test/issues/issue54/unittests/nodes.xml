<?xml version="1.0" encoding="utf-8"?>
<nodes version="1.0" xmlns:ding="http://zanstra.com/ding">
  <ding:dong-dang.dung>Namespaced element with dash and dot</ding:dong-dang.dung>
  <yo123-ho456/>
  <yo123-ho457>data</yo123-ho457>
  <nil/>
  <empty></empty>
</nodes>
