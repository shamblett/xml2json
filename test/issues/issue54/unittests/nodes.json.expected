{"nodes":{"ding:dong-dang.dung":"Namespaced element with dash and dot","yo123-ho456":null,"yo123-ho457":"data","nil":null,"empty":null}}
